recipe/meta.yaml

```
@@ -1,46 +1,67 @@
 {% set name = "fast-histogram" %}
-{% set version = "0.9" %}
-{% set sha256 = "6325ce42060bb9d1987a3aefffcac45bdd3810bc58ce69716132e23aa49f7a9f" %}
+{% set version = "0.14" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
+  sha256: 390973b98af22bda85c29dcf6f008ba0d626321e9bd3f5a9d7a43e5690ea69ea
 
 build:
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  #[py<39]
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
     - python
+    - wheel
     - setuptools
     - setuptools_scm
     - pip
-    - numpy
+    - numpy >=2.0.0
   run:
     - python
     - {{ pin_compatible('numpy') }}
```

```suggestion
    - numpy
```
The numpy pinned in the cbc has run exports.